browse/controllers/__init__.py

```diff
@@ -51,3 +50,9 @@ def biz_tz() -> ZoneInfo:
         return _arxiv_biz_tz
     else:
         return _arxiv_biz_tz
+
```

Comment:
Minor: what happens if the old_keys has a key that is in the new keys?

Reply:
fastly ignores duplicates, but i can fix that